server/http_hooks.go

```diff
@@ -229,6 +231,37 @@ func (p *Plugin) handleEphemeralDelete(w http.ResponseWriter, r *http.Request) {
 	_, _ = w.Write(resp.ToJson())
 }
 
+func (p *Plugin) handleInteractionAction(w http.ResponseWriter, r *http.Request) {
+	request := model.PostActionIntegrationRequestFromJson(r.Body)
+	if request == nil {
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	user, appErr := p.API.GetUser(request.UserId)
+	if appErr != nil {
+		p.API.LogError("failed to get user for interaction action", "err", appErr.Error())
+		w.WriteHeader(http.StatusOK)
+		return
+	}
+
+	requestJSON, _ := json.MarshalIndent(request, "", "  ")
```

Comment:
Please check the error.

Reply:
Oops, copied this from line 54 and did not modify thereafter, will do.